recipes/desc_bpz/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "DESC_BPZ" %}
+{% set version = "0.3.2" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/desc_bpz-{{ version }}.tar.gz
+  sha256: eb145abf3d7a326f0b817dee87e8c069d27517c659720485722b5a4c323bb180
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=62
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - scipy
+    - pandas >=1.1
+    - h5py
+
+test:
+  imports:
+    - desc_bpz
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}  
+    - pip
+
+about:
+
+  home: http://github.com/LSSTDESC/{{ name }}
```

Comment:
Please don't use jinja variables in the URLs, it obscures them and doesn't really help the recipe.

Reply:
sure, FWIW, that is from `grayskull`